.githubmap

```diff
@@ -38,7 +38,7 @@ ukernel Zheng Yan <zyan@redhat.com>
 vasukulkarni Vasu Kulkarni <vasu@redhat.com>
 vuhuong Vu Pham <vuhuong@mellanox.com>
 wjwithagen Willem Jan Withagen <wjw@digiware.nl>
-xiexingguo xie xingguo <xie.xingguo@zte.com.cn>
```

Comment:
I believe this has something to do with my `Signed-off-by: ` line.
@theanalyst right?

Reply:
@xiexingguo Yes, .githubmap is used to find the Name and Email ID of reviewers from the github userid, to mention them during the merge of a PR.

Reply:
@xiexingguo Do you think that the current entry in .githubmap is correct ? If so, I will close this PR.

Reply:
@joscollin, I'd leave it as it is (at least to keep pace with my `Signed-off-by:` line). 
Thanks all the same for pointing this out to me!

Reply:
@xiexingguo No Problem. I found this difference by accident, while searching for your name and email for adding it to the `Reviewed-by:` line during a merge. Later on, I figured out that the current entry is matching with your `Signed-off-by`. I will close the PR now. Thanks for confirming.